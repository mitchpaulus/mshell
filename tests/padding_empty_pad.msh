"cat" "" 5 leftPad wl
