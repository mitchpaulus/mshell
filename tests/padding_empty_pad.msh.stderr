1:12: Cannot leftPad with an empty pad string.
